sdk/eventhub/event-processor-host/changelog.md

```diff
@@ -1,6 +1,10 @@
-### 2019-07-24 2.1.0
+### 2019-07-30 2.1.0
 - Added support for WebSockets. WebSockets enable Event processor Host to work over an HTTP proxy and in environments where the standard AMQP port 5671 is blocked.
 Refer to the [websockets](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/eventhub/event-processor-host/samples/websockets.ts) sample to see how to use WebSockets. 
+- Fixed [issue](https://github.com/Azure/azure-sdk-for-js/issues/4363)
+  - Removed `_eitherLeaseManagerOrleaseDurationAndRenewal(options)` method. 
+    - If a LeaseManager is provided then leaseDuration and leaseRenewInterval values in the options will be ignored.
+    - If nothing is provided(leaseDuration and leaseRenewInterval or leaseDuration and leaseRenewInterval from the LeaseManager) then we will set defaults.
```

Comment:
`_eitherLeaseManagerOrleaseDurationAndRenewal` is an internal function and the user has no knowledge of it. Therefore mentioning that we removed it is of no use to our users. I would suggest the below

> Fixed [bug 4363](https://github.com/Azure/azure-sdk-for-js/issues/4363) which stopped users from providing their own LeaseManager. If both a lease manager and the options for leaseDuration/leaseRenewInterval are provided, then the latter will be ignored in favor of the leaseDuration/leaseRenewInterval properties on the lease manager.